src/main/java/com/kodilla/ecommercee/domain/OrderItem.java

```diff
@@ -0,0 +1,45 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.jetbrains.annotations.NotNull;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@Entity(name = "ORDER_ITEM")
+public class OrderItem {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
skąd te adnotacje?

Reply:
I added this annotation with `strategy...` because Witold add it too, but he chose `strategy = GenerationType.IDENTITY` with which I disagree.  I think it should be `strategy = GenerationType.AUTO` and in point of fact we don't need to write anything more (annotation `@GeneratedValue` is enough), because if the strategy is not specified by default AUTO will be used. I just used this to mark out that it should be `Auto` not `Identity`.